src/Text/Pandoc/Writers/MediaWiki.hs

```diff
@@ -372,6 +372,10 @@ inlineToMediaWiki (Emph lst) = do
   contents <- inlineListToMediaWiki lst
   return $ "''" <> contents <> "''"
 
+inlineToMediaWiki (Underline lst) = do
+  contents <- inlineListToMediaWiki lst
+  return $ "<u>" <> contents <> "</su>"
```

Comment:
Is `</su>` correct here? Should it be `</u>`?

Reply:
Oops, thanks for pointing this out!